src/services/userService.ts

```diff
@@ -0,0 +1,38 @@
+//import { createUser, getUserById, getUsers, updateUser } from '../data-access/sequelizeORMDataAccess';
+import { createUser, getUserById, getUsers, updateUser } from '../data-access/typeORMDataAccess';
+import { User, UserUpdates } from '../types';
+
+export class UserService {
+  public async createUser(user: User) {
+    const result = await createUser(user);
+    return result;
```

Comment:
You can refactor it like this:
```suggestion
    return await createUser(user);
```

Reply:
fixed